### ISSUE TITLE
Tests with [E]rrors don't show test name

### ISSUE THREAD

**Issue by JSteitz** [LimeDeck/phpunit-detailed-printer#2]
Tests that fail because of an error in the test will not show the test name.
If the test follows another one it will show the name from the previous test.

```
✔ Tests\Integration\Jobs\PrepareVRDownloadTest: Could not get signature for device id (713 ms)
! Tests\Integration\Jobs\PrepareVRDownloadTest: Could not get signature for device id (713 ms)
✔ Tests\Integration\Jobs\PrepareVRDownloadTest: Copied signature for device id (765 ms)
```

**Comment by jakubhomoly** [comment]
Thanks for your report. We're working on this issue, and we're going to release a new version with this fixed.

**Comment by jakubhomoly** [comment]
This has been fixed in the version 2.0. Thanks for your patience!
